recipe/meta.yaml

```diff
@@ -10,7 +10,7 @@ source:
 
 build:
   number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  script: {{ PYTHON }} -m pip install . -vv
```

Comment:
This is already done for us by conda-build on [UNIX]( https://github.com/conda/conda-build/blob/7816bc881b937e46be1d7ca70285bb3c7b44bb4a/conda_build/build.py#L3054-L3074 ) and [Windows]( https://github.com/conda/conda-build/blob/7816bc881b937e46be1d7ca70285bb3c7b44bb4a/conda_build/windows.py#L296-L317 )